### DISCUSSION TITLE
[Q] Is roadrunner supported on the aarch64 architecture?

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [roadrunner-server/roadrunner#774] [Q&A] [answered]
Is roadrunner supported on the aarch64 architecture? I have an AWS server setup `t4g.small` and I am unable to run roadrunner on this. Following is the error that I get.

```
ERROR  sh: 1: exec: /home/forge/myapp.url.com/rr: Permission denied
```

**Comment by rustatian** [comment, marked as answer]
You showed an error that is not related to the architecture. You should just tune the permissions for the binary. 
Regarding the initial question, **Yes**, we support `arm64` platform. You can download the related binary from the releases repository: https://github.com/spiral/roadrunner-binary/releases
